try.py

```diff
@@ -13,11 +13,9 @@
     print()
 
 # try opening a device, then perform write and read
-
+h = hid.device()
 try:
```

Comment:
Moved h out of the try clause to be able to access it in the except clause.

This shouldn't be a problem since hid.device() doesn't throw anything